.circleci/config.yml

```diff
@@ -353,19 +328,10 @@ workflows:
             branches:
               ignore:
               - /docs-preview/
-      - build-bazel:
-          requires:
-            - build
       - test:
           requires:
             - build
```

Comment:
I believe `test` can depend only on `setup` since it doesn't use any of the legacy artifacts

Reply:
The legacy artifacts aren't used, but the build job is very quick to run (~35s) and will catch any build errors before proceeding to the more lengthy and resource intensive Bazel jobs. 

If they were to run in parallel and there was a build error, all three would just show the same build errors but take longer to fail and use up more CircleCI credits.

Reply:
Makes sense to me. Perhaps leave a comment that this is intentional?

Reply:
SGTM, added.